src/AcceptanceTesting/SpecByExample/SFA.DAS.Payments.AcceptanceTests/Features/Ilr Changes/learner_changes_contract_type.feature

```diff
@@ -1,188 +0,0 @@
-﻿@learner_changes_contract_type
```

Comment:
I thought we were leaving the test code in?

Reply:
not the features for 965. Just the framework updates which we have since built more specs on.

Reply:
I don't fully understand why 965 is reverting but I'm guessing it's temporary?? if so we can just ignore these scenarios for now.